pca/test/test_XMLData/tokens.xml

```diff
@@ -1,8 +1,307 @@
 <tokens>
-	<!-- Test entries likely need to be added before testing will work -->
 	<entry>
-			<token>0123456789ab</token>
+			<token>04ea957ec11c</token>
 			<email>test1@example.org</email>
-			<cid>2800000a-500e-4008-a00e-600000000007</cid>
+			<cid>28fc34fa-51be-4518-a7be-656cbdf01ae7</cid>
```

Comment:
Why did you change the token and cid values?  I just want to make sure there isn't some leakage here.

Reply:
The token and cid were unnecessarily sanitized when they were moved from the private repository to this public one.  I changed them back so that they align with the values in the rest of the test data. There is no sensitive info here.

Reply:
Coolio - I'd sanitized them just-in-case without the context of whether they were sensitive or not. Glad to hear they aren't!